src/docs/devices/Buderus-KM271-WiFi/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Buderus KM271-WiFi
+date-published: 2025-01-15
+type: misc
+standard: global
+board: esp32
+project-url: https://github.com/the78mole/ESPhome-KM271-WiFi
+made-for-esphome: True
```

Comment:
No application has been received via email
```suggestion
made-for-esphome: False
```

Reply:
Thanks :-)

Reply:
Hi @jesserockz, sorry for asking, but I don't understand, what I shall do? Sending the application to you via Email?
BTW: My HW is made for esphome... You can download the Release from the repo.

Reply:
https://esphome.io/guides/made_for_esphome